### ISSUE TITLE
Entry point script

### ISSUE THREAD

**Issue by ilectra** [UCL/kaska#32]
Moving this to its own issue, as we currently have a different entry point script in each branch, soon all to be merged into master.

Currently 
- `develop` runs kaska in each tile and then combines them (in parallel using dask if available)
- `sar_wcm` runs kaska_sar on the whole image
- `entry_point` runs kaska on the whole image, but contains the right way to define a script in a python package
